doc/howtos/building-images.rst

```diff
@@ -122,6 +122,26 @@ with ``# require conf/distro/include/ostro-os-development.inc`` and
 uncomment it.
 
 
+Accelerating Build Time Using Shared-State Files Cache
+------------------------------------------------------
+
+As explained in the `Yocto Project Shared State Cache documentation`_, by design
+the build system builds everything from scratch unless it can determine that
+parts do not need to be rebuilt. The Yocto Project shared state code supports
+incremental builds and attempts to accelerate build time through the use
+of prebuilt data cache objects configured with the ``SSTATE_MIRRORS`` setting.
+
+By default, this ``SSTATE_MIRRORS`` configuration is disabled in :file:`conf/local.conf`
+but can be easily enabled by uncommenting the ``SSTATE_MIRRORS`` line
+in your :file:`conf/local.conf` file, as shown here:::
+
+   # Example for Ostro OS setup, recommended to use it:
+   #SSTATE_MIRRORS ?= "file://.* http://download.ostroproject.org/sstate/ostro-os/PATH"
```

Comment:
On Thu, 2016-03-10 at 05:30 -0800, Gustavo Sverzut Barbieri wrote:
> eSDK AFAIU also uses SSTATE for othe bite, just the native tools comes
> from different path, coming in the SDK itself and is modified to be
> relocatable and host-agnostic

True. But my point was more about our documentation: do we still need to
document SSTATE_MIRRORS and how to use them directly? Is that still
relevant once we start using the eSDK?

> How to solve the host-agnostic coming from SSTATE is still to be
> solved, @kad is talking to Yocto folks. 

eSDK does something regarding that with uninative.bbclass, but
apparently there are still issues left.